src/components/CTA/CTA.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FunctionComponent } from 'react';
+
+import classnames from 'classnames';
+
+import { DefaultProps } from '../../types';
+import { HeadingType } from '../../types/index';
+import { Button } from '../Button/Button';
+import { Heading } from '../Heading/Heading';
+import { IconName } from '../Icon/types';
+import { RichText } from '../RichText/RichText';
+
+import './CTA.scss';
+
+export interface CTAProps extends DefaultProps {
+	heading: string;
+	headingType: HeadingType;
+	content: string | string[];
+	buttonLabel: string;
+	buttonIcon?: IconName;
```

Comment:
Wat gaat er gebeuren als we op een button klikken?
mss nog een onClick optie toevoegen

Dit zal normaal altijd een interne of externe link zijn. Maar daar kunnen we niet naar navigeren in de components aangezien react-router hier niet geïmplementeerd is, dus dit zelf de "block preview" component moeten regelen in de client